test/runtests.jl

```diff
@@ -2,6 +2,14 @@
 
 include("choosetests.jl")
 tests, net_on = choosetests(ARGS)
+tests = unique(tests)
```

Comment:
Since it was easiest to write the special-casing of the `compile.jl` test in a way that removed only a single instance from the list, I figured that for correctness and paranoia I should call `unique(tests)`.  But yes, it leaves the order as-is and should ordinarily be a no-op.